File: 2015/gara2/medaglie/att/medaglie.pas
function gioca(N: longint; var M: array of integer): longint;
begin
    (* Mettete qui il codice della soluzione *)
    gioca := 42;
end;


const
    MAXN = 100000;
var
    N, i  : longint;
    M     : array[0..MAXN-1] of integer;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);
    for i:=0 to N-1 do
        read(fr, M[i]);

    writeln(fw, gioca(N, M));
    close(fr);
    close(fw);
end.

File: ed1/gara2/medaglie/att/medaglie.c
#include <stdio.h>
#include <assert.h>

#define MAXN 100000

int gioca(int N, short M[]) {
    // Mettete qui il codice della soluzione
    return 42;
}


short M[MAXN];

int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));
    for(i=0; i<N; i++)
        assert(1 == fscanf(fr, "%hd", &M[i]));

    fprintf(fw, "%d\n", gioca(N, M));
    fclose(fr);
    fclose(fw);
    return 0;
}
